ui/accordion.js

```diff
@@ -361,7 +361,10 @@ return $.widget( "ui.accordion", {
 			maxHeight = 0;
 			this.headers.next()
 				.each( function() {
+					var display = $( this ).css( "display" );
+					$( this ).css( "display", "block" );
```

Comment:
Well, more explicitly, follow this logic: https://github.com/jquery/jquery-ui/blob/adcc8eff431847b4897666369310cb83275a8282/ui/dialog.js#L634-L642